card/lib/cardio.rb

```diff
@@ -50,7 +50,7 @@ def default_configs
         paging_limit:           20,
 
         non_createable_types:   [%w(signup setting set)],
-        view_cache:             true,
+        view_cache:             false,
```

Comment:
If we change this, we have to make sure we change the config on staging/production.

I think I still favor defaulting to having it on, but I understand why you'd want it off.